ffi/wasm/README.md

```diff
@@ -0,0 +1,123 @@
+# @boa-dev/boa_wasm
+
+WebAssembly bindings for [Boa](https://github.com/boa-dev/boa), an experimental JavaScript lexer, parser and interpreter written in Rust.
```

Comment:
```suggestion
WebAssembly bindings for [Boa](https://github.com/boa-dev/boa), a Javascript engine written in Rust.
```

We're planning to release 1.0 in the near future, so we can remove the "experimental" wording for the next release.

Reply:
Uhh this wasn't resolved...